bundle/regal/ast.rego

```diff
@@ -402,6 +402,24 @@ is_chained_rule_body(rule, lines) if {
 	startswith(col_text, "{")
 }
 
+# METADATA
+# description: |
+#   returns whether a keyword is imported in the policy, either explicitly
+#   like "future.keywords.if" or implicitly like "future.keywords" or "rego.v1"
+imports_keyword(imports, keyword) if {
+	some imp in imports
+
+	_has_keyword(arr(imp), keyword)
+}
+
+arr(xs) := [y.value | some y in xs.path.value]
```

Comment:
hmm looking at this now, we could just inline `arr(.)`, or promote it to some utility method. But it's fine either way, let's keep it.